ci/taos/checker-pr-audit-async.sh

```diff
@@ -233,8 +233,9 @@ popd
 
 # Pull-Request scheduler for Tizen build (gbs)
 # The default RUN Queue is declared in the configuration file
-while [ `ps aux | grep "sudo.*gbs build" | wc -l` -gt $RUN_QUEUE_PR_JOBS ]; do
-    WAITTIME=$(( ( RANDOM % 20 ) + 20 ))
+current_jobs=$(ps aux | grep "sudo.*gbs build" | wc -l)
+while [ $current_jobs -gt $RUN_QUEUE_PR_JOBS ]; do
```

Comment:
current_jobs doesn't changed. so it can make infinite loop

Reply:
Updated. resubmitted.